countrypicker/src/main/java/com/canopas/campose/countrypicker/CountyUtils.kt

```diff
@@ -32,7 +32,9 @@ fun getJsonDataFromAsset(context: Context, fileName: String): String? {
 fun List<Country>.searchCountryList(countryName: String): MutableList<Country> {
     val countryList = mutableListOf<Country>()
     this.forEach {
-        if (it.name.lowercase().contains(countryName.lowercase())) {
+        if (it.name.lowercase().contains(countryName.lowercase()) ||
+            it.dial_code.lowercase().contains(countryName.lowercase())
```

Comment:
Is it required to use lowercase() in dial code?
Because what i think is dial code will be having numeric value only

Reply:
Can you throw some light on this @cp-luzi-p ?

Reply:
@cp-hardik-p when user search county using dial code bottom sheet get not any value that's why i added this.